tests/system/libraries/NvdaLib.py

```diff
@@ -196,6 +225,15 @@ def runKeyword(*args, **kwargs):
 			)
 		return remoteLib
 
+	def start_NVDAInstaller(self, settingsFileName):
+		builtIn.log(f"Starting NVDA with config: {settingsFileName}")
+		self.setup_nvda_profile(settingsFileName)
+		nvdaProcessHandle = self._startNVDAInstallerProcess()
+		process.process_should_be_running(nvdaProcessHandle)
+		self._connectToRemoteServer(connectionTimeoutSecs=30)

```

Comment:
This extra timeout is probably due to the start up splash sound. It might be worth a comment here to highlight that it is different from the usual NVDA startup timeout.